src/sentry/snuba/metrics/fields/base.py

```diff
@@ -141,12 +138,43 @@ def run_metrics_query(
         + where,
         granularity=Granularity(GRANULARITY),
     )
+
     request = Request(
-        dataset=Dataset.Metrics.value,
+        dataset=Dataset.Metrics.value
```

Comment:
For some reason, the wrong dataset was used but we still got data back, interesting.